### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by OshekharO**
Added many open source Application's and some tutorials

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,6 +113,10 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**Kotatsu**](https://github.com/nv95/Kotatsu) <sup>**[[F-Droid](https://f-droid.org/app/org.koitharu.kotatsu)]**</sup>
 * [**Nekome**](https://github.com/Chesire/Nekome) <sup>**[[F-Droid](https://f-droid.org/app/com.chesire.nekome)]**</sup>
 * [**Tachiyomi**](https://github.com/tachiyomiorg/tachiyomi)
+* [**Kayuri**](https://github.com/Killerpac/Kayuri)
+* [**Animity**](https://github.com/kl3jvi/animity)
+* [**Saikou**](https://github.com/saikou-app/saikou)
+* [**Kotatsu**](https://github.com/nv95/Kotatsu)
```
Comment (offa) [inline README.md:119]: It's included already (some lines above) :-)

File: README.md [hunk 2]
```diff
@@ -113,6 +113,10 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**Kotatsu**](https://github.com/nv95/Kotatsu) <sup>**[[F-Droid](https://f-droid.org/app/org.koitharu.kotatsu)]**</sup>
 * [**Nekome**](https://github.com/Chesire/Nekome) <sup>**[[F-Droid](https://f-droid.org/app/com.chesire.nekome)]**</sup>
 * [**Tachiyomi**](https://github.com/tachiyomiorg/tachiyomi)
+* [**Kayuri**](https://github.com/Killerpac/Kayuri)
+* [**Animity**](https://github.com/kl3jvi/animity)
+* [**Saikou**](https://github.com/saikou-app/saikou)
```
Comment (offa) [inline README.md:118]: These two include proprietary google components (eg. firebase).

File: README.md [hunk 3]
```diff
@@ -331,6 +335,9 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**Jellyfin**](https://github.com/jellyfin/jellyfin-android) <sup>**[[F-Droid](https://f-droid.org/app/org.jellyfin.mobile)]**</sup>
 * [**Twire**](https://github.com/twireapp/Twire) <sup>**[[F-Droid](https://f-droid.org/app/com.perflyst.twire)]**</sup>
 * [**LibreTube**](https://github.com/libre-tube/LibreTube) <sup>**[[F-Droid](https://f-droid.org/app/com.github.libretube)]**</sup>
+* [**VueTube**](https://github.com/VueTubeApp/VueTube)
```
Comment (offa) [inline README.md:338]: From a first look it seems to me that Google Services are optional here. Does anybody have more information?

F-Droid is considered btw.: https://github.com/VueTubeApp/VueTube/issues/40 :+1:

File: README.md [hunk 4]
```diff
@@ -331,6 +335,9 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
...
+* [**SmartTubeNext**](https://github.com/yuliskov/SmartTubeNext)
```
Comment (offa) [inline README.md:339]: It contains quite some proprietary and tracking libraries importunately: https://github.com/yuliskov/SmartTubeNext/issues/471#issuecomment-858083371

The comment is from last year, but the situation seems still the same.

File: README.md [hunk 5]
```diff
@@ -331,6 +335,9 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
...
+* [**PsTube**](https://github.com/prateekmedia/pstube)
```
Comment (offa) [inline README.md:340]: :+1: Should be on F-Droid in the near future: 

- https://github.com/prateekmedia/pstube/issues/19
- https://gitlab.com/fdroid/rfp/-/issues/2002

File: README.md [hunk 6]
```diff
@@ -638,12 +647,15 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
...
 * [**TWRP**](https://twrp.me)
+* [**OrangeFox**](https://orangefox.download)
+* [**PitchBlackRecovery**](https://pitchblackrecovery.com)
 
 ------------------------
 
 ## – Tutorials and Guides –
 
 * [**OTA on Rooted Device**](tutorials/Install-Rooted-OTA.md)
+* [**Reverse Engineering**](https://github.com/OshekharO/Reverse-Engineering/wiki)
```
Comment (offa) [inline README.md:658]: I'm not sure if this really fits a Open Source list.

File: README.md [hunk 7]
```diff
@@ -655,3 +667,4 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**CellularPrivacy – Recommendations**](https://github.com/CellularPrivacy/Android-IMSI-Catcher-Detector/wiki/Recommendations)
 * [**Awesome Android**](https://github.com/TheEvilSkeleton/awesome-android)
 * [**List of free and open-source Android applications**](https://en.wikipedia.org/wiki/List_of_free_and_open-source_Android_applications)
+* [**Cloudflare Reverse Proxy**](https://github.com/OshekharO/CF-REVERSE-PROXY)
```
Comment (offa) [inline README.md:670]: Same as above.

**Review by offa — CHANGES_REQUESTED** [review]
Thank you! :-)

I have left some review comments.

File: README.md [hunk 8]
```diff
@@ -404,6 +412,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**SongTube**](https://github.com/SongTube/SongTube-App)
 * [**raag**](https://github.com/raag-music/raag) <sup>**[[F-Droid](https://f-droid.org/app/in.amfoss.raag)]**</sup>
 * [**Music**](https://github.com/z-huang/music)
+* [**ViMusic**](https://github.com/vfsfitvnm/ViMusic)
```
Comment (offa) [inline README.md:415]: Addendum: Let's go for F-Droid :crossed_fingers: https://github.com/vfsfitvnm/ViMusic/issues/32

File: README.md (hunk 6 above)
Comment (OshekharO) [reply README.md:658]: Yup it does fit in  tutorials and guide

**Pull request closed without merging**